ambry-api/src/main/java/com.github.ambry/config/ReplicationConfig.java

```diff
@@ -98,6 +98,14 @@
   @Default("1048576")
   public final long replicationFetchSizeInBytes;
 
+  /**
+   * If true, replication Get requests asks for deleted and expired blobs as well to succeed in scenarios where blobs
+   * get deleted or expired after replication metadata exchange.
+   */
+  @Config("replication.include.all")
```

Comment:
can we just generalize this and have this receive a `GetOption` instead of `true`/`false`?

Reply:
Is this supposed to be `true` or `false` by default?

Reply:
I think generalizing this is unnecessary. Once we validate the flow, I plan to get rid of this config.

Default `true` is what I intend for this to be eventually, so `true` by default.